go/chat/attachment.go

```diff
@@ -114,7 +129,7 @@ func putSingle(ctx context.Context, log logger.Logger, r io.Reader, size int64,
 	// attachment.  But putSingle is only called for attachments <= 5MB, so
 	// this isn't horrible.
 	buf := make([]byte, size)
-	n, err := r.Read(buf)
+	n, err := io.ReadFull(r, buf)
```

Comment:
Yes, probably correct.  It doesn't seem like it can hurt to leave the check in, but if you'd like I can remove it.